diff --git a/src/database/schema.rs b/src/database/schema.rs
index 78a8686..2346fd1 100644
--- a/src/database/schema.rs
+++ b/src/database/schema.rs
@@ -13,7 +13,7 @@ diesel::table! {
         ingredient_id -> Integer,
         calories -> Float,
         quantity -> Float,
-        quantity_units -> Nullable<Text>,
+        quantity_units -> Nullable<crate::database::models::IngredientMeasurementMapping>,
     }
 }
 
@@ -23,7 +23,7 @@ diesel::table! {
         recipe_id -> Integer,
         ingredient_id -> Integer,
         quantity -> Float,
-        quantity_units -> Nullable<Text>,
+        quantity_units -> Nullable<crate::database::models::IngredientMeasurementMapping>,
     }
 }
 
@@ -47,7 +47,7 @@ diesel::table! {
         id -> Integer,
         name -> Text,
         description -> Text,
-        duration -> Text,
+        duration -> crate::database::models::RecipeDurationMapping,
         category -> Integer,
     }
 }
